pkg/containerd/importer/importer.go

```diff
@@ -60,6 +60,22 @@ func isLayerTar(name string) bool {
 	return slashes == 2 && strings.HasSuffix(name, "/layer.tar")
 }
 
+// followSymlinkLayer returns actual layer name of the symlink layer.
+// It returns "deadbeeddeadbeef/layer.tar" if the name is like
+// "../deadbeeddeadbeef/layer.tar", and returns error if the name
+// is not in "../xxxx/layer.tar" format.
```

Comment:
../foo/layer.tar not xxxx unless you want to use xxxx as the replacement for deadbeeddeadbeef & deadbeefdeadbeef strings...